src/converter.js

```diff
@@ -148,19 +149,16 @@ var getQueryStrings = function(swagger, path, method, values) {
 
   if (typeof swagger.paths[path][method].parameters !== 'undefined') {
     for (var i in swagger.paths[path][method].parameters) {
-      var param = swagger.paths[path][method].parameters[i]
+      var param = swagger.paths[path][method].parameters[i];
       if (typeof param['$ref'] === 'string' &&
         !/^http/.test(param['$ref'])) {
         param = resolveRef(swagger, param['$ref'])
       }
       if (typeof param.in !== 'undefined' && param.in.toLowerCase() === 'query') {
+        const sample = OpenAPISampler.sample(param.schema || param, {});
         queryStrings.push({
           name: param.name,
-          value: typeof values[param.name] === 'undefined'
-            ? (typeof param.default === 'undefined'
-              ? ('SOME_' + (param.type || param.schema.type).toUpperCase() + '_VALUE')
-              : param.default + '')
-            : (values[param.name] + '') /* adding a empty string to convert to string */
+          value: JSON.stringify(sample)
```

Comment:
You shouldn't ignore `values` arguments and just use `sample` as fallback values.